### DISCUSSION TITLE
Using nuxt-directus with a static Nuxt site

### DISCUSSION THREAD

**Discussion by miket2marcom** [Intevel/nuxt-directus#170] [Q&A]
Hi,

We use nuxt-directus on a number of our Nuxt sites with Directus backends, and it's a great package.

Our Nuxt sites are fully static - generated with `npx nuxi generate` - and the behavior of these sites with nuxt-directus is not exactly what we are expecting. The other day, we had an issue where one of our Directus cloud backends went down for a few hours, and during this time, the site's pages were not loading.

How do we configure this so that, upon build time, all content is pulled from Directus and put into the static site? The idea here is for the generated static site to be fully independent of anything that's going on with the Directus backend. Basically we'd like our static builds to remain immutable, with all Directus content "inline", unless and until we trigger a new build for future updates/changes.

Thanks for your help here,
Mike

**Comment by Intevel** [comment]
Hey @miket2marcom, checkout this Issue here: https://github.com/Intevel/nuxt-directus/issues/81
